trl/trainer/policy_trainer_base.py

```diff
@@ -313,6 +305,17 @@ def __exit__(self, exc_type, exc_value, traceback):
             self.optional_peft_ctx.__exit__(exc_type, exc_value, traceback)
 
 
+# PR TODO: move this experiment somewhere if successful
+def faster_inference_model(model):
+    model.generation_config.cache_implementation = "static"
```

Comment:
I think this only works with llama and not mistral

Reply:
Yep, saw no improvement on my end. Thanks for the context!